style.css

```diff
@@ -0,0 +1,74 @@
+header {
+  background-image: url(assets/header-background-mobile.png);
+  position: absolute;
+  width: 375px;
+  height: 100px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo a {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  padding: 10px 12px;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.headline {
+  background-image: url(assets/pic1.png);
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+}
```

Comment:
- [ ] Kindly add these properties to the headline class to make the background image match the design in Figma
```
    background-repeat: no-repeat;
    background-size: contain;
```